### PR TITLE
Add sortevent

### PR DESCRIPTION
**Pull request by venuslimm**
This commit consists of
- Renaming of `FindEventTest.java` to `FindEventCommandTest.java`
- Changing of `dateTime`'s datatype (variable in DateTime class) from `String` to `LocalDateTime`
- Adding `sortevent` command
  - Users are able to sort events based on the event's name (ascending/descending order) or its start date time (ascending order) or its end date time (ascending order)
  - Sorting does not persist
  - Tests are added for the codes written for this feature

### CODE REVIEW

File: src/main/java/seedu/address/model/event/exceptions/SortComparatorException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package seedu.address.model.event.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Persons (Persons are considered duplicates if they have the same
+ * identity).
+ */
```
Comment (jerome-neo) [inline src/main/java/seedu/address/model/event/exceptions/SortComparatorException.java:6]: JavaDoc needs correction for SortComparatorException

[the remark this answers is not in the document]
Comment (venuslimm) [reply src/main/java/seedu/address/model/event/exceptions/SortComparatorException.java]: Corrected! Thank you for pointing out

File: src/main/java/seedu/address/logic/commands/SortEventCommand.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        if (model.getFilteredEventList().size() > 1) {
+            model.sortEventList(sortEventType);
+            return new CommandResult(String.format("%s based on the %s", MESSAGE_SUCCESS,
+                    sortEventType.getDescription()));
+        } else {
+            return new CommandResult("Too few events in the current list to sort");
```
Comment (inas0ng) [inline src/main/java/seedu/address/logic/commands/SortEventCommand.java:43]: Content of message could be written as a constant for ease of editability in the future, for example,
`public static final String MESSAGE_TOO_FEW_EVENTS = "Too few events in the current list to sort"`

**Review by inas0ng — COMMENTED** [review]
code is very organised with good use of enums! i left 1 minor suggestion for improvement

File: src/main/java/seedu/address/logic/commands/SortEventCommand.java (hunk 2 above)
Comment (venuslimm) [reply src/main/java/seedu/address/logic/commands/SortEventCommand.java:43]: Thanks, I have corrected it!

**Review by inas0ng — APPROVED** [review]

**Review by Irminrics — APPROVED** [review]
LGTM!

**Pull request merged**